lib/logatron/railtie.rb

```diff
@@ -95,7 +79,25 @@ class Railtie < Rails::Railtie
         app.middleware.use Logatron::Middleware
       end
     end
-
-    SetupLograge.setup(config)
+    config.after_initialize do
+      SetupLograge.setup(config)
+    end
   end
 end
+
+if defined? Lograge::Railtie
+  raise [
+        'Lograge::Railtie cannot be required before Logatron::Railtie. ',
+        'The config.after_initialize of Logatron must run before that ',
+        'of Lograge::Railtie.'
+    ].join
+end
+begin
+  require 'lograge'
+rescue LoadError
+  msg = [
+      'Include "lograge" in your bundle. It was left out of the gemspec of',
+      'Logatron to keep dependencies down for projects that do not use Rails.'
```

Comment:
I assume the problem dependency chain starts with `lograge` -> `railties` ?

Reply:
I am uncertain what you mean by this.

Reply:
don't worry about it.